frontend/src/components/AppointmentCard.tsx

```diff
@@ -0,0 +1,38 @@
+import './AppointmentCard.scss';
+import {useContext, useEffect, useState} from "react";
+import axios from "axios";
+import {AuthContext} from "../context/AuthProvider";
+
+
+export default function AppointmentCard() {
+
+    const [appointments, setAppointments] = useState([])
+
+    const {token} = useContext(AuthContext)
+
+    useEffect(() => {
+        axios.get("/api/appointments", token? {
+            headers: {
+                "Authorization": token
+            }
+        } : {})
+            .then(response => {
+                console.log(response)
+                setAppointments(response.data)
+            })
+            .catch(console.error)
```

Comment:
okay for now, but maybe add some error handling logic

Reply:
yeah, the handling logic will be implemented later :)